extensions/VisualEditor/modules/ve-mw/init/targets/ve.init.mw.ViewPageTarget.js

```diff
@@ -986,6 +982,12 @@ ve.init.mw.ViewPageTarget.prototype.saveDocument = function ( saveDeferred ) {
 	}
 };
 
+ve.init.mw.ViewPageTarget.prototype.isSummaryMissing = function ( saveOptions ) {
```

Comment:
This method name is misleading. Can we think of better name?
For example: `shouldShowMissingSummaryMessage`